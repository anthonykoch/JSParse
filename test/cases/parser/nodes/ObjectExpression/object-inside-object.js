{
    lives: {
        address: {
            zip: 90210,
            near: {
                place: 'somewhere only we know'
            }
        }
    }
}
