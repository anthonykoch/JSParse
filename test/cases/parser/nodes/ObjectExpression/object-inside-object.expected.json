---
name: ObjectExpression - object inside object
description: nested object
skip: true
---

